recipes/typhos/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "typhos" %}
+{% set version = "0.7.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/pcdshub/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: cdcc30a6e1a7eeb72f2a0b9db711dd28cbfc073e3c39e9ac2b7bfaa7fbb31821
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  entry_points:
+    - typhos = typhos.cli:main
+    - typhon = typhos.cli:main
```

Comment:
What is the difference between `typhos` and `typhon` ?

Reply:
Ah, I see the package for renamed. Is that why?

Reply:
Yes, this is something that will be removed in the next version, but remains here for back-compatibility for now.